### DISCUSSION TITLE
Hocus: apply a style to both hover and focus.

### DISCUSSION THREAD

**Discussion by trevyn** [tailwindlabs/tailwindcss#2325] [Ideas]
Would love to have a config setting to add this. :)

**Comment by trevyn** [comment]
Oh wait, is this not possible with plain CSS? Got this from twin, but that rewrites everything.

**Comment by paulpopus** [comment]
`:hocus` doesn't exist in plain CSS if you're referring to [this](https://www.npmjs.com/package/postcss-hocus). But you can easily write your own plugin for it if you want.

I may write an example one if/when I get some time.
